utilities/GeppettoConfiguration.json

```diff
@@ -1,6 +1,6 @@
 {
     "_README" : "http://docs.geppetto.org/en/latest/build.html",
-    "contextPath": "org.geppetto.frontend",
+    "contextPath": "org.geppetto.frontend/",
```

Comment:
I modified 2 lines in **org.geppetto.frontend**  to make this line work:

- https://github.com/openworm/org.geppetto.frontend/pull/859/commits/b52f41afda3db9b7e743b27afc3a236ed4ec880e
- https://github.com/openworm/org.geppetto.frontend/pull/859/commits/9bb8912696747fbaaa5d3773ba0c697c6dbd825b

I think **org.geppetto.frontend** needs to move away from things like this:
- host = path1 + path2 + path3

And go with something like this:
- host = some_function(path1, path2, path3)